### PR TITLE
Aut5 hostname

### PR DESCRIPTION
**Pull request by shinitiandrei**
Added cookbook chef_hostname
Added unit and integration tests
Added folder attributes
Minor fixes and refactoring

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -9,8 +9,18 @@
...
+AllCops:
+  TargetRubyVersion: 2.3
+
 Metrics/LineLength:
   Enabled: false
 
 Style/RegexpLiteral:
   Enabled: false
+
+Style/AccessorMethodName:
```
Comment (igorlg) [inline .rubocop.yml]: @shinitiandrei Why this?

File: spec/unit/recipes/config_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#
+# Cookbook Name:: chef-base
+# Spec:: default
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+require 'spec_helper'
+
+describe 'chef-base::config' do
+  PLATFORMS.each do |p|
```
Comment (igorlg) [inline spec/unit/recipes/config_spec.rb]: We're taking down multiple platform support for now, please remove the platform loop...

File: test/integration/default/serverspec/default_spec.rb [hunk 3]
```diff
@@ -59,3 +59,10 @@
     it { expect(stdout_err).to match %r{Vi IMproved 7.4} }
   end
 end
+
+describe 'hostname is set' do
+  describe command 'hostname' do
```
Comment (igorlg) [inline test/integration/default/serverspec/default_spec.rb]: use syntax: command**(**'hostname'**)** do

File: .rubocop.yml (hunk 1 above)
Comment (shinitiandrei) [inline .rubocop.yml]: I needed to add the "set" method, but rubocop does not allowed me to do it, saying that's not a good "name" for it.

Comment (igorlg) [inline .rubocop.yml]: Cool, no problem =]

File: spec/unit/recipes/config_spec.rb (hunk 2 above)
Comment (shinitiandrei) [inline spec/unit/recipes/config_spec.rb]: yes, changed

File: test/integration/default/serverspec/default_spec.rb (hunk 3 above)
Comment (shinitiandrei) [inline test/integration/default/serverspec/default_spec.rb]: ok, changed

**Pull request merged**